foundation/foundation/src/main/res/values/attrs.xml

```diff
@@ -39,9 +39,22 @@
     <attr name="vtmnBorderPrimaryReversed" format="color" />
 
     <!-- States -->
+    <attr name="vtmnActivePrimary" format="color" />
+    <attr name="vtmnActivePrimaryTransparent" format="color" />
+    <attr name="vtmnActiveSecondaryTransparent" format="color" />
+    <attr name="vtmnActiveTertiary" format="color" />
+    <attr name="vtmnActiveTertiaryTransparent" format="color" />
     <attr name="vtmnActiveBrand" format="color" />
     <attr name="vtmnActiveAccent" format="color" />
+    <attr name="vtmnActivePrimaryReversedTransparent" format="color" />
+    <attr name="vtmnActiveSecondaryReversedTransparent" format="color" />
+    <attr name="vtmnActiveTertiaryReversedTransparent" format="color" />
+    <attr name="vtmnActiveBrandReversedTransparent" format="color" />
 
     <!-- Typography -->
     <attr name="textAppearanceBody3" format="reference" />
+    <attr name="textAppearanceBody1Bold" format="reference" />
+    <attr name="textAppearanceBody2Bold" format="reference" />
+    <attr name="textAppearanceBody3Bold" format="reference" />
```

Comment:
We really need to add this? We can't just apply a bold style on the Body{number] text appearance?

Reply:
After a discussion with designer, you can remove this.